### ISSUE TITLE
Game Launcher Rofi 

### ISSUE THREAD

**Issue by RookiexCookie** [prasanthrangan/hyprdots#887]
### Description

So Hey , I tried using the Hyprdots game launcher but whichever style i chose  there was something kinda weird about the icons for ex not properly displaying the thumbnail, not proper padding 
it felt like its for 2k monitors   

For style 1 i expected 
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/b160d6c3-b7da-403b-8876-c83d70d031f7)
but i got this
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/897e7235-1c7e-44c4-8839-97c771af24b6)

For Style 2 
i got something like this 
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/88ff23e3-2530-4cb3-a487-0a0f08981185)

For Style 3
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/107ded67-f7b9-447f-8530-f064559ae7d9)

For Style 4 
i was getting like this 
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/e7cd67f9-46b5-40f5-98cd-aea686838e15)
but i was able to tinker around the columns and etc to make it normal
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/05a4bbc5-40a5-45b2-b91e-9430e592974a)

and For style 5 ooh boy
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/0652a5a2-4c20-4e71-a0af-0a533299dbfb)
cuz the config has this
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/368c83a8-bb07-4ebf-86cd-b5284317bb2b)

#---------------
oh and my resolution is    1366x768

**Comment by T-Crypt** [comment]
@RookiexCookie have you tried changing the width and the height in the config to reflect your screen res? 

Try : 

```
width;                                    1375px;
height:                                   700px; 
```

You can try modifying the width and height to fit as you like. You will need to restart rofi with pkill rofi for changes.

**Comment by RookiexCookie** [comment]
Hello @T-Crypt , Ive messed around with columns and padding with one style and i got that one  somewhat how it was meant to be, however this last one is kinda weird
after your advise ,its like this now
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/b3f34eae-3e70-4836-8895-a347589004fb)

i mean its not important anyways
but i just wanted to let you guys know.

also one thing
in your hyprdots theme windows11 and mac the toolbar is black , how do i make em white

**Comment by Enigma1309** [comment]
Try copying the file from repo and modify, **only** just these values,
```
width;                                    1375px;
height:                                   700px; 
```

**Comment by RookiexCookie** [comment]
@Enigma1309 same results
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/df47ced1-d560-4760-b213-a88c79081a53)

**Comment by Enigma1309** [comment]
there's a new pr related to the lutris integration, that might work out in this case #1011 
tldr try out the new gamelauncher.sh with the pr and follow [these](https://github.com/prasanthrangan/hyprdots/pull/1011#issuecomment-1976508210) steps (the comment is marked off topic but it works)

**Comment by kRHYME7** [comment]
Hey sorry I removed the part that dynamically adjusts the image ratio to fit on the monitor(intended on a separate PR).
